packages/react-core/src/components/Button/Button.tsx

```diff
@@ -20,6 +21,16 @@ export enum ButtonType {
   submit = 'submit',
   reset = 'reset'
 }
+
+export interface BadgeCountObject {
+  /**  Adds styling to the badge to indicate it has been read */
+  isRead?: boolean;
+  /** Adds count number right of button */
+  count?: number;
+  /** Additional classes added to the Badge */
```

Comment:
nit - sentence case. Though FWIW these classes aren't added to the badge (component) - they're added to `.pf-c-button__count`, which is a structural element of the button, so it isn't entirely accurate that the classes are added to the badge. I say that since, kind of like we're adding a prop to `<Button>` to pass props/classes to `.pf-c-badge__count` since the count element isn't exposed, it's fair to assume we would offer a prop to pass props/classes to `<Badge>`, too.

```suggestion
  /** Additional classes added to the badge */
```